rfcs/react-components/non-core-components.md

```diff
@@ -0,0 +1,388 @@
+- 👎 Our partners might still want to customize to the max
+- 👎 Approaches could be heavily biased on a subset of partners
+
+### Breaking change cadence
+
+#### Same cadence as core components
+
+Breaking changes for non-core components will follow the same cadence as core components i.e.
+breaking changes can only happen when @fluentui/react-components reaches v10.
+
+##### Pros
+
+- 👍 Predictability - nothing new compared to current library
+- 👍 Good support for partners to build on
+
+##### Cons
+
+- 👎 Partners might not mind breaking changes for specialized components that are not used often
+- 👎 Maintaining legacy for components that are not used often in applications
+
+#### Faster than core components
+
+Breaking changes for non-core components can occur more frequently than for core components. The proposal does not
+mean that breaking changes will be taken lightly, or that there will be a requirement to make breaking changes often.
+
+##### Pros
+
+- 👍 Bleeding edge improvements for partners
+- 👍 Specialized components might not have widespread use and can be upgraded in a few instances
+
+##### Cons
+
+- 👎 Cost of breaking changes has not changed
+- 👎 Just because we make breaking changes does not mean users will upgrade earlier
+- 👎 Need to handle urgent bugfixes for most past versions - need a clear support SLA
+
+### Choosing a repo for
+
+#### Current monorepo
+
+Non-core packages should still live in the monorepo.
+
+##### Pros
+
+- 👍 Easier setup
+- 👍 Nothing new to learn
+- 👍 Issue reporting path does not change
+
+##### Cons
+
+- 👎 We might not want the same release cadence - releasing a fix for a non-core component would release all of core
+- 👎 Extra load on the CI as more non-core components are needed
+- 👎 Not possible to have more separation from core components without lots of tooling hacks
+
+#### Separate monorepo
+
+We can create a new monorepo for our non-core packages and depend on core packages from NPM.
+
+##### Pros
+
+- 👍 We can have our ideal DX
+- 👍 Release cadence does not need to depend on core
+- 👍 Developing our with our public API - we eat our own dogfood
+
+##### Cons
+
+- 👎 Needs effort to setup
+- 👎 Issue reporting becomes more unclear
+- 👎 Requires linking to use latest core code
+- 👎 Manual/Automated bumps required to keep packages up to date
+
+### Package scope
+
+#### @fluentui scope
+
+Non-core packages would be published under the same `@fluentui/` NPM scope.
+
+##### Pros
+
+- 👍 No need to create and manage another organization
+- 👍 Easier discoverability for OSS users and internal partners
+
+##### Cons
+
+- 👎 Not clear that there is a difference between non-core components and core components/utilities
+- 👎 Harder for us to tell the difference between non-core and core
+
+#### New package scope
+
+Creates a new package scope, `@fluentui-banana/` (name pending) for all components/packages that are non-core.
```

Comment:
new scope might make sense only when its out of FUI domain - eg system that builds on core controlls but provides completely different UI